toolkit/tools/internal/rpm/rpm_test.go

```diff
@@ -199,3 +201,56 @@ func TestExtractNameFromRPMPath(t *testing.T) {
 		})
 	}
 }
+
+func TestDistroDefines(t *testing.T) {
```

Comment:
Would it be possible to test one scenario per test function? It's going to be clearer what we're checking for and what exactly failed in case of a failure.

Reply:
Basically, instead of a comment line for each test case in this one function, my suggestion is to have a separate test function with a name that reflects what it's testing.